docs/integration/new-window.md

```diff
@@ -22,8 +22,8 @@ your application.
 To integrate our offsite panel you need to add the following tags to your application HTML, replacing <pilot_id> (e.g. hackstage-pass)
 and <panel_type> (e.g. link, rating) with the appropriate values before embedding.  
 
-    <script src="www.bbc.co.uk/taster/offsite-panel.js" type="text/javascript" />
-    <iframe class="taster-offsite-panel" src="www.bbc.co.uk/taster/projects/<pilot_id>/offsite/<panel_type>" frameborder="0"></iframe>
+    <script src="http://www.bbc.co.uk/taster/offsite-panel.js" type="text/javascript" />
```

Comment:
I don't have write access to this repo, but this `<script>` tag shouldn't be self-closing here, too (eg. needs a closing `</script>` tag).